packages/key-management/package.json

```diff
@@ -47,6 +47,7 @@
     "prepack": "yarn build"
   },
   "devDependencies": {
+    "@cardano-sdk/tx-construction": "^0.2.1",
```

Comment:
I think when you do a relative import then you need to update test project tsconfig
```
"include": [
    "../../../tx-construction/test/testData"
  ],
```
This dependency doesn't feel right, I'd remove it from package.json - can still use it with "includes"

Reply:
Done and squashed fcfa035a3498f675945dafcc82b8f05c08318dd8